### DISCUSSION TITLE
idea中点击此插件时的窗口初始化大小能不能大一些？

### DISCUSSION THREAD

**Discussion by 2565911151** [YiiGuxing/TranslationPlugin#5277] [Ideas]
使用的时候点击此插件弹出对话框感觉太小了，如果翻译的内容较多还得手动拖拽调整大小感觉很麻烦，建议设置一个合适的初始窗口大小，或者设一个可以自定义的设置，让用户自定义窗口初始大小。

**Comment by YiiGuxing** [comment]
翻译对话框的大小是有记忆功能的，调整其大小后关闭，下次打开会恢复上一次关闭时的大小的
